appveyor.yml

```diff
@@ -0,0 +1,9 @@
+version: '4.5.4-SNAPSHOT+AppVeyor.{build}'
+os: Windows Server 2012
+build_script:
+  - mvn compile -q --batch-mode 
```

Comment:
:question: Does AppVeyor come with Maven installed? If not, you can install it using Chocolatey [like this](https://github.com/sharwell/antlr4cs/blob/master/build/appveyor-install.ps1#L11) followed by setting `M2_HOME` [like this](https://github.com/sharwell/antlr4cs/blob/master/appveyor.yml#L7).